### ISSUE TITLE
Fine Tuning

### ISSUE THREAD

**Issue by mzahran001** [fmassa/object-detection.torch#21]
How to fine tune your model?
I don't have sufficient data to retrain your model from scratch.I want to fine tune your model on my data which has only two classes ?

**Comment by fmassa** [comment]
Hi,

If you don't have enough data, I'd advise you to train an SVM using the last fc7 layer from the network.
There is a [SVM trainer in here](https://github.com/fmassa/object-detection.torch/blob/refactoring2/SVMTrainer.lua).

Using it will require you to convert your dataset to have labels in a format similar to Pascal VOC 2012 (or to write a dataset that has the same interface as [`DatasetPascal`](https://github.com/fmassa/object-detection.torch/blob/refactoring2/DataSetPascal.lua), which also includes computing bounding boxes.

**Comment by brisker** [comment]
@fmassa  Hi, after I trained a model from scratch, I found that the epoch seems not enough, so I reload the model and try to finetune it for a bit more epoches. But why the training loss seems like training from scratch?? I just use this code:
model=torch.load(...)
local parameters, gradParameters = model:getParameters()
...
The other codes are just the same as the code for training from scratch. Why this happens? I am really confused about this.
